src/wfqc/network.py

```diff
@@ -230,40 +201,30 @@ def create_citation_network(topicID="topic_0121", testSize='', randomSeed=42, lo
             os.mkdir(outpath)
 
         # Downloading data
-        edges, nr_citations, included_tools = download_data(outpath,testSize,randomSeed, topicID)
+        edges, included_tools = asyncio.run(download_data(outpath,testSize,randomSeed, topicID))
 
         # Creating the graph using igraph
         print("Creating citation graph using igraph.")
 
-        G, included_tools, node_degree_dict, unq_edges =  create_graph(edges, included_tools, includeCitationNodes = includeCitationNodes)
+        G, included_tools, node_degree_dict =  create_graph(edges, included_tools)
 
       
         # Saving edges, graph and tools included in the graph 
         if saveFiles:
-            edge_path = f'{outpath}/edges{testSize}.pkl'
             graph_path = f'{outpath}/graph{testSize}.pkl'
-            tool_path = f'{outpath}/tools{testSize}.pkl'
-            nr_citations_path = f'{outpath}/nr_citations{testSize}.pkl'
             node_degree_dict_path = f'{outpath}/node_degree_dict{testSize}.pkl'
 
             print(f"Saving data to directory {outpath}.") # sould make these filenames dynamic
             # and save them 
             #Do this nicer later? TODO: perhaps save them all in a json instead
-            with open(edge_path, 'wb') as f:
-                pickle.dump(unq_edges, f)
 
             with open(graph_path, 'wb') as f:
                 pickle.dump(G, f)
 
-            with open(tool_path, 'wb') as f:
-                pickle.dump(included_tools, f)  
-
-            with open(nr_citations_path, 'wb') as f:
-                pickle.dump(nr_citations, f)
-
             with open(node_degree_dict_path, 'wb') as f:
                 pickle.dump(node_degree_dict, f)
 
 
     # returns a graph and the pmids of the tools included in the graph (tools connected by cocitations)
-    return G, included_tools 
+    return G
```

Comment:
I would suggest to rename `G` to something more descriptive, in general single letter variable names are not the best practice (unless we are talking about counters (`i`, `j`), coordinates (`x`,`y`), etc. You can leave `f` (line 224) as it clearly represents a file, but in general it's not a good practice. I see another one later in the code, I will leave the comment there as well.